project/Dependencies.scala

```diff
@@ -13,6 +13,7 @@ object Dependencies {
   val junitVersion = "0.11"
   val graphvizVersion = "0.2.2"
   val directoryWatcherVersion = "0.4.0"
+  val monixVersion = "2.3.0"
```

Comment:
You might want to give 3.0.0-RC3 a try, see https://github.com/monix/monix/releases/tag/v3.0.0-M3

> One other highlight of the release are the performance improvements for Task and Coeval, an on-going process to make sure that Monix's Task remains the best implementation in Scala's ecosystem.

Reply:
I'll benchmark the two, thanks for the suggestion.